### PR TITLE
NMS-14791 Topology-Map Layer broken

### PR DESCRIPTION
**Pull request by rssntn67**
Fixed DefaultVertex in enlinkd topology updaters
    Let IplikeSearchProvider be available for all nodes related namespaces
    Fixed IpLikeHopCriteria getVertices getting vertices from the graph namespace

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14791

### CODE REVIEW

File: features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/support/IpLikeHopCriteria.java [hunk 1]
```diff
@@ -78,12 +88,14 @@ public IPVertex(String id) {
...
     }
 
-	public IpLikeHopCriteria(SearchResult searchResult, IpInterfaceProvider ipInterfaceProvider) {
+	public IpLikeHopCriteria(SearchResult searchResult, IpInterfaceProvider ipInterfaceProvider, GraphContainer graphContainer) {
     	super(searchResult.getQuery());
     	m_collapsed = searchResult.isCollapsed();
         m_ipQuery = searchResult.getQuery();
         this.ipInterfaceProvider = Objects.requireNonNull(ipInterfaceProvider);
         m_collapsedVertex = new IPVertex(m_ipQuery);
+		m_graphContainer = Objects.requireNonNull(graphContainer);
```
Comment (aramosvizcarra) [inline features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/support/IpLikeHopCriteria.java:97]:
```suggestion
	m_graphContainer = Objects.requireNonNull(graphContainer);
```

File: features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/support/IpLikeHopCriteria.java [hunk 2]
```diff
@@ -78,12 +88,14 @@ public IPVertex(String id) {
...
+		Objects.requireNonNull(graphContainer.getTopologyServiceClient());
```
Comment (aramosvizcarra) [inline features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/support/IpLikeHopCriteria.java:98]:
```suggestion
	Objects.requireNonNull(graphContainer.getTopologyServiceClient());
```

**Review by aramosvizcarra — CHANGES_REQUESTED** [review]
just a couple of formatting changes.

**Review by christianpape — APPROVED** [review]
LGTM.

**Review by aramosvizcarra — APPROVED** [review]

**Pull request merged**